doc/config.yml

```diff
@@ -31,6 +31,7 @@ Extensions:
     MooseDocs.extensions.tagging:
         active: True
         js_file: index-6fae072b.js
+        csv_file: summary.csv
```

Comment:
@GiudGiud looks like this is causing the documentation test to fail

Reply:
ah too early then. Let s remove it

Reply:
Removed